env/CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ ExternalProject_Add(
     GIT_REPOSITORY https://github.com/llvm/llvm-project.git
     GIT_TAG ${LLVM_PROJECT_VERSION}
     GIT_PROGRESS ON
+    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/build.patch
+    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/generated.patch
+    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/mathextras.patch
+    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/toolchains.patch
+    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/zstd.patch
```

Comment:
@AleksKnezevic, are these patches really necessary for building XLA?  My understanding is that we're going to integrate with Bazel, so XLA will build our SW inside of their build of llvm.

Reply:
Yeah, good point, these shouldn't be necessary.